### ISSUE TITLE
resize_nearest_neighbor.h:37:32: error: 'round' is not a member of 'std'

### ISSUE THREAD

**Issue by 408881465** [squix78/esp32-tensorflow-microspeech#1]
Hi there,
When I build this project, It shows errors below.

Compiling .pio\build\esp-wrover-kit\libaa5\tfmicro\tensorflow\lite\micro\kernels\softmax.cc.o
Compiling .pio\build\esp-wrover-kit\libaa5\tfmicro\tensorflow\lite\micro\kernels\split.cc.o
In file included from lib\tfmicro\tensorflow\lite\micro\kernels\resize_nearest_neighbor.cc:16:0:
lib\tfmicro/tensorflow/lite/kernels/internal/reference/resize_nearest_neighbor.h: In function 'int32 tflite::reference_ops::GetNearestNeighbor(int, int32, int32, bool, bool)':
lib\tfmicro/tensorflow/lite/kernels/internal/reference/resize_nearest_neighbor.h:37:32: error: 'round' is not a member of 'std'
           ? static_cast<int32>(std::round((input_value + offset) * scale))
                                ^
lib\tfmicro/tensorflow/lite/kernels/internal/reference/resize_nearest_neighbor.h:37:32: note: suggested alternative:
In file included from c:\users\40888\.platformio\packages\toolchain-xtensa32\xtensa-esp32-elf\include\c++\5.2.0\cmath:44:0,
                 from lib\tfmicro/tensorflow/lite/kernels/internal/reference/resize_nearest_neighbor.h:18,
                 from lib\tfmicro\tensorflow\lite\micro\kernels\resize_nearest_neighbor.cc:16:
C:\users\******\.platformio\packages\framework-arduinoespressif32\tools\sdk\include\newlib/math.h:278:15: note:   'round'
 extern double round _PARAMS((double));
               ^
*** [.pio\build\esp-wrover-kit\libaa5\tfmicro\tensorflow\lite\micro\kernels\resize_nearest_neighbor.cc.o] Error 1
